### PR TITLE
4.get/api/articles

### PR DESCRIPTION
**Pull request by dan77uk**

### CODE REVIEW

File: __tests__/getArticles.test.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+const seed = require("../db/seeds/seed");
+
+beforeEach(() => {
+  return seed(data);
+});
+afterAll(() => {
+  return db.end();
+});
+
+describe("GET /api/topics", () => {
```
Comment (emilyb93) [inline __tests__/getArticles.test.js]: I think you may have written the wrong end point in this describe here

File: models/selectArticles.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+              return Promise.reject({
+                status: 204,
+                msg: "No Content",
+              });
+            }
+            return result.rows;
+          });
+      });
+    });
+};
```
Comment (emilyb93) [inline models/selectArticles.js]: nested promise chain, move this into a single chain please

**Review by emilyb93 — CHANGES_REQUESTED** [review]
Un-nest your promise chain

File: __tests__/getArticles.test.js (hunk 1 above)
Comment (dan77uk) [reply __tests__/getArticles.test.js]: Corrected

**Review by hevdevs — APPROVED** [review]
Looks good to merge based on suggested changes!

**Pull request merged**